src/assets/scripts/client/scope/RadarTargetModel.js

```diff
@@ -239,8 +243,8 @@ export default class RadarTargetModel {
      * @chainable
      */
     _initializeScratchPad() {
-        if (!this.aircraftModel.destination) {
-            this._scratchPadText = 'XXX';
+        if (this.aircraftModel.isDeparture()) {
+            this._scratchPadText = this.aircraftModel.fms.getFlightPlanEntry();
```

Comment:
This will need to be limited to 3 characters and upper-cased, probably best to achieve by adding a setter which does a `text.substr(0, 3).toUpperCase()`.

Also, two lines down will need `.substr(1, 3).toUpperCase()` in order to show `SEA` in the data block instead of `sea`.